darwin/importer/importer.py

```diff
@@ -378,4 +418,13 @@ def _import_annotations(
     if append:
         payload["overwrite"] = "false"
 
+    if delete_for_empty and not annotations:
```

Comment:
We should add `payload["overwrite"] = "true"` in all other cases where we don't set `false`. We seem to have a bug on BE side that doesn't properly default to `true`, which makes this impossible to test this PR unless we default to `true` on darwin-py side.

Reply:
Done !